### ISSUE TITLE
Investigate the dynamic configuration of Kafka plugins

### ISSUE THREAD

**Issue by sknot-rh** [strimzi/strimzi-kafka-operator#3325]
More info: https://github.com/strimzi/strimzi-kafka-operator/pull/2389#discussion_r414670947

**Comment by scholzj** [comment]
The dynamic configuration currently handles only Kafka configuration options. We need to investigate how to handle dynamic configuration for different Kafka plugins which support it, but are not part of the main Kafka configuration model - such as authorisers etc.

**Comment by scholzj** [comment]
Triaged on 31.3.2022: Jakub will recheck if it is feasible in a reasonable.

**Comment by scholzj** [comment]
Sounds like some more work would need to be done in Kafka first to support this in all kind of plugins. Even once that is done, it might not make it easy for Strimzi to decide what are all the configuration options the plugin is suing since it might also consume some other configuration. We would also need to add the configuration infromation for "supported" plugins to the config models probably.

**Comment by scholzj** [comment]
Triaged on 12.4.2022: Needs proper support in Kafka first. Nothing to do in Strimzi in the mean time. Can be closed.
